test/Profiler/unmapped.swift

```diff
@@ -93,10 +93,10 @@ struct UnavailableType {
   var storedClosure: Int = { .random() ? 0 : 1 }()
 
   @Wrapper
-  var wrappered = .random() ? 0 : 1
+  var wrapped = .random() ? 0 : 1
```

Comment:
I'm probably being too paranoid, but the identifier `wrapped` is, in some really specific cases, special for property wrappers. I don't think it means anything special *here*, but out of an abundance of caution I'd be more comfortable giving this a name that completely avoids that problem, like `wrappedProperty`.